agent/php_api.c

```diff
@@ -144,12 +146,14 @@ PHP_FUNCTION(newrelic_notice_error) {
     case 5:
       if (FAILURE
           == zend_parse_parameters_ex(ZEND_PARSE_PARAMS_QUIET,
-                                      ZEND_NUM_ARGS() TSRMLS_CC, "lsslz!",
-                                      &ignore1, &errormsgstr, &errormsglen,
-                                      &ignore2, &ignore3, &ignore4, &ignore5)) {
+                                      ZEND_NUM_ARGS() TSRMLS_CC, "lssls!",
```

Comment:
The format for the parameters changes so the last value is a 's' instead of 'z' - what is the rationale behind this change?

Reply:
This has been changed to a 's' because in the [docs API](https://docs.newrelic.com/docs/apm/agents/php-agent/php-agent-api/newrelic_notice_error/#syntax), it mentions that `error_context` is a string which is the 5th parameter for this function.